### PR TITLE
Redesign hero banner

### PR DESCRIPTION
**Pull request by hdJerry**
As an user of the application I would like my navbar to be fully redesigned, so that the side has a Web 3 look and feel

NOTE: the branch need to be merged into a new branch called "redesign" otherwise it will break the live site!

AC:
- [x] Add the new background image and make sure that it also cover the navbar
- [x] Chang the words and placement of the headings
- [x] Change the look and feel of the CTA button

Check figma for latest designs:
image

Screenshot from 2022-07-25 12-15-16

### CODE REVIEW

File: src/sections/HomePageBanner/HomePageBanner.module.css [hunk 1]
```diff
@@ -1,156 +1,52 @@
...
-@media (min-width: 1476px) {
-  .imageContainer:before {
-    left: 7rem;
-  }
-
-  .image {
-    margin-left: 4.1875rem;
-  }
+  justify-content: center;
+  box-shadow: 0px 18px 22px rgba(255, 255, 255, 0.08), 0px 7.51997px 9.19107px rgba(255, 255, 255, 0.0575083), 0px 4.02054px 4.91399px rgba(255, 255, 255, 0.0476886), 0px 2.25388px 2.75474px rgba(255, 255, 255, 0.04), 0px 1.19702px 1.46302px rgba(255, 255, 255, 0.0323114), 0px 0.498106px 0.608796px rgba(255, 255, 255, 0.0224916);
```
Comment (Zelig880) [inline src/sections/HomePageBanner/HomePageBanner.module.css:51]: can you approximate the PX of this box shadow please to make this more readable.. (so if it is 7.51997px change to 7px)

Also not sure if we can do the same with the colors transparency. To be honest, this is not really adding a strong shadow, so change values slightly is not going to affect this much!

File: src/sections/HomePageBanner/HomePageBanner.module.css [hunk 2]
```diff
@@ -1,156 +1,52 @@
 .container {
-  background-color: var(--secondary-200-color);
-  padding: 0 2rem;
-}
-
-.banner,.banner_left {
-  margin: auto;
+  background-image:  var(--bg-gradient), url(/images/hero-bg.png);
```
Comment (Zelig880) [inline src/sections/HomePageBanner/HomePageBanner.module.css:2]: can you please also add the BODY background color with this PR please.

File: src/styles/globals.css [hunk 3]
```diff
@@ -29,6 +29,7 @@ body {
   scroll-padding-top: 6rem;
 }
 
+
```
Comment (Zelig880) [inline src/styles/globals.css:32]: as you are doing other actions can you also remove this extra line, not needed

**Review by Zelig880 — CHANGES_REQUESTED** [review]
Just a few comments.

File: src/styles/globals.css (hunk 3 above)
Comment (hdJerry) [reply src/styles/globals.css:32]: Okay, will do!

**Review by Zelig880 — CHANGES_REQUESTED** [review]
The navbar cannot be clicked! it is behind the hero!

**Review by Zelig880 — APPROVED** [review]

**Pull request merged**